packages/ref/src/cmip_ref/cli/datasets.py

```diff
@@ -43,8 +44,17 @@ def list_(
     data_catalog = adapter.load_catalog(database, include_files=include_files, limit=limit)
 
     if column:
-        if not all(col in data_catalog.columns for col in column):
-            logger.error(f"Column not found in data catalog: {column}")
+        missing = set(column) - set(data_catalog.columns)
+        if missing:
+
+            def format(columns: Iterable[str]) -> str:
+                return ", ".join(f"'{c}'" for c in sorted(columns))
+
+            logger.error(
+                f"Column{'s' if len(missing) > 1 else ''} "
+                f"{format(missing)} not found in data catalog. "
+                f"Choose from: {format(data_catalog.columns)}"
+            )
```

Comment:
```suggestion
            def _format(columns: Iterable[str]) -> str:
                return ", ".join(f"'{c}'" for c in sorted(columns))

            logger.error(
                f"Column{'s' if len(missing) > 1 else ''} "
                f"{_format(missing)} not found in data catalog. "
                f"Choose from: {_format(data_catalog.columns)}"
            )
```

Reply:
Well spotted! I fixed it and enabled the ruff rule that warns about this: https://docs.astral.sh/ruff/rules/#flake8-builtins-a in a6b365a.